src/qt/askpassphrasedialog.cpp

```diff
@@ -145,11 +158,7 @@ void AskPassphraseDialog::accept()
                                      tr("The supplied passphrases do not match."));
             }
         }
-        else
-        {
-            QDialog::reject(); // Cancelled
-        }
-        } break;
+    } break;
```

Comment:
When originally testing, the reject() seemed redundant, because the window was also closing without this line. Since the next thing to happen was the method finishing / returning I assumed at that time that the default behaviour of QDialog class was being triggered. But now I was having second thoughts, and brought back the original behavior of calling an explicit reject.